### PR TITLE
Bug 1191745 - Switch to new homescreen by default. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a5b7d8e9e2b6065de5e1b8495e2e4eca8b950453

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=716d9a15c6213c51716f0d38c8278aacd4c60e48

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f20dace2afd4de8942fab2fc55ddb5103a47cb7b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1d7b941223a9e99abcd1a8fc393b3e7bc25d184

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b724a90f66a15d716cb9265af6d801e655a1e509

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=604dde7d80f04c75b113d6336baf7e6bd2f20780

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31a7703f5a9fc6f869556a678ffeb85e22713538

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83e71ca2372ae6be876c43059a558d622625dace

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bbceb1f7d904f2f87c56b4d82367f2c47f2fd9bb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b1616c050deebc97135985c73fa1977a7a1cdf8c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6dc55a10face91e93ddc9f656b35d100d510790a

File: apps/verticalhome/manifest.webapp [hunk 1]
```diff
@@ -24,7 +24,7 @@
       "description": "Gaia الشاشة الرئيسية"
     },
     "en-US": {
-      "name": "Default Home Screen",
+      "name": "Legacy Home Screen",
       "description": "Gaia Home Screen."
```
Comment (gmarty) [inline apps/verticalhome/manifest.webapp]: Do we need to translate this in other languages too?

File: apps/homescreen/manifest.webapp [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "New Home Screen",
+  "name": "Default Home Screen",
```
Comment (TheoChevalier) [inline apps/homescreen/manifest.webapp]: In order to make this string localizable, we should do the same as in https://github.com/mozilla-b2g/gaia/blob/master/apps/verticalhome/manifest.webapp#L21

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4ed1c9421b1ec6f7bd1fad758be6d3aff0bc10ab

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7ef852e00234822dd7a50ca72593df5d052f453c

**Pull request merged**